testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -152,13 +152,35 @@
 # Check a checkbox of the given id
 #
 When(/^I check "([^"]*)"$/) do |identifier|
-  check(identifier)
-  raise ScriptError, "Checkbox #{identifier} not checked." unless has_checked_field?(identifier)
+  begin
+    contenedor = find('div', text: identifier, exact_text: true)
```

Comment:
What does `contenedor` mean? I assume it is Spanish, so please use the english translation :wink: